src/python/pants/option/options_test.py

```diff
@@ -1455,8 +1484,10 @@ def parse_joined_command_line(*args):
             ParseError,
             dedent(
                 """\
-                Unrecognized command line flags on scope 'compile': --modifylogs. Suggestions:
-                --modifylogs: [--compile-scala-modifylogs]"""
+                Unrecognized command line flag '--modifylogs' on scope 'compile'. Suggestions:
+                --compile-scala-modifylogs
```

Comment:
Nice! Would consider indenting the suggestion flag because it prevents it from looking like it wrapped from the previous line.